application/src/test/java/org/opentripplanner/updater/trip/siri/AddedTripBuilderTest.java

```diff
@@ -551,7 +551,7 @@ void testAddedTripFailOnUnknownStop() {
 
     assertTrue(addedTrip.isFailure(), "Trip creation should fail");
     assertEquals(
-      UpdateError.UpdateErrorType.NO_VALID_STOPS,
+      UpdateError.UpdateErrorType.UNKNOWN_STOP,
```

Comment:
There is a test for UNKNOWN_STOP here but perhaps you could add one for regular, non-extra-journey updates.